docs/swagger.json

```diff
@@ -0,0 +1,234 @@
+                "parameters": [
+                    {
+                        "description": "The external ID of a task status record.",
+                        "in": "path",
+                        "name": "uuid",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Information about the status of a user-triggered task.",
+                        "examples": {
+                            "application/json": {
+                                "name": "Export of course 'Data Structures 101'",
+                                "state": "Succeeded",
+                                "state_text": "成功しました",
+                                "completed_steps": 4,
+                                "total_steps": 4,
+                                "attempts": 1,
+                                "artifacts": [
+                                    "https://www.example.com/artifacts/ef48901f-2277-4b51-9589-9e834277b23c/"
+                                ]
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "There is no such task status record (or the user lacks permission to view it)."
+                    }
+                },
+                "tags": [
+                    "tasks"
+                ]
+            }
+        },
+        "/tasks/{uuid}/cancel/": {
+            "post": {
+                "summary": "Cancel the task associated with the specified status record.",
+                "description": "Revoke the task if it has not yet been started, instruct it to halt at the next convenient stopping point if already running, or do nothing if it has already stopped running.",
+                "parameters": [
+                    {
+                        "description": "The external ID of a task status record.",
+                        "in": "path",
+                        "name": "uuid",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "201": {
+                        "description": "Confirmation that cancellation of the task is being processed."
+                    },
+                    "403": {
+                        "description": "The user is allowed to view this status record, but lacks permission to cancel it."
+                    },
+                    "404": {
+                        "description": "There is no such task status record (or the user lacks permission to view it)."
+                    }
+                },
+                "tags": [
+                    "tasks"
+                ]
+            }
+        }
+    },
+    "produces": ["application/json"],
+    "schemes": ["https"],
+    "swagger": "2.0"
+}
```

Comment:
Yes, this gets auto-generated as `docs/rest_api.rst` (which isn't checked in since it's a build artifact).  That's done by `swg2rst` from the `swagger2rst` package in the `docs` tox environment.